spec/models/podcast_spec.rb

```diff
@@ -0,0 +1,12 @@
+require "rails_helper"
+
+RSpec.describe Podcast, type: :model do
+  let(:podcast) { create(:podcast) }
+
+  it "accepts valid podcast and busts cache" do
```

Comment:
It's not clear who accepts a podcast :) These also seem like 2 separate specs.
The suggestion is the same as in the above specs.

Reply:
I updated the specs like above.